### PR TITLE
Second match test cases

### PR DESCRIPTION
**Pull request by colawwj**

### CODE REVIEW

**Review by qiaozha — APPROVED** [review]
LGTM

File: .config/CredScanSuppressions.json [hunk 1]
```diff
@@ -4,6 +4,14 @@
         {
             "placeholder": "Aa!1()-xyz",
             "_justification": "This is the secret place holder used by my team"
+        },
```
Comment (lirenhe) [inline .config/CredScanSuppressions.json:7]: Do we really need to create a couple of different secret?  Can we use one placeholder in all code?

Comment (qiaozha) [reply .config/CredScanSuppressions.json:7]: Different service has different requirements for secret.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "azure-samples-for-js-management",
+  "version": "1.0.0",
```
Comment (lirenhe) [inline package.json:3]: As the SDK is in preview, so for the samples, shall we also mark them as preview?  @nickzhums

File: package.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "eslint . --ext .ts --ignore-pattern node_modules/ --ignore-pattern dist/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure-Samples/azure-samples-js-management.git"
+  },
+  "author": "",
```
Comment (lirenhe) [inline package.json:14]: Do we have an alias for JS SDK?

**Comment by qiaozha** [comment]
@colawwj  Could you add links for sql/web/keyvault in https://github.com/Azure-Samples/azure-samples-js-management/blob/main/README.md like this.
image

**Review by lirenhe — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (nickzhums) [reply package.json:3]: what is this name for?

Comment (qiaozha) [reply package.json:3]: This is just the package name for this sample repo.

File: package.json (hunk 3 above)
Comment (qiaozha) [reply package.json:14]: @nickzhums  Could you take a look at this question ?

Comment (nickzhums) [reply package.json:14]: do not think so

**Pull request merged**